pkgs/applications/video/upscayl/default.nix

```diff
@@ -0,0 +1,35 @@
+{appimageTools, fetchurl, lib}:
+
+let
+  pname = "upscayl";
+  version = "2.0.1";
+  src = fetchurl {
+    url = "https://github.com/upscayl/upscayl/releases/download/v${version}/upscayl-${version}-linux.AppImage";
+    sha256 = "sha256-qsFYWUK5GwuiAaU7Y3tAM8Eea1G+R/MwoDSFUZvo/cM=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit pname version src; };
+in
+appimageTools.wrapType2 rec {
+  inherit pname version src;
+
+  name = pname;
+
```

Comment:
```suggestion
```
that would remove version

Reply:
This is required so that the executable name is upscayl and not upscayl-2.0.1

Reply:
You want to rename the executable instead of overwriting name.

Reply:
This must be changed